test/Makefile

```
@@ -220,6 +220,7 @@ auto-tester-test:
 else
 auto-tester-test: $(RUNNER)
 	$(EXECUTE_RUNNER) runnable compilable fail_compilation dshell
+	$(EXECUTE_RUNNER) --unit-tests
```

Include the `unit` tests into the first invocation:

```suggestion
	$(EXECUTE_RUNNER) unit_tests runnable dshell compilable fail_compilation
```